### ISSUE TITLE
Warning: unrecognized parameter Style/AlignParameters:IndentationWidth

### ISSUE THREAD

**Issue by donv** [bbatsov/rubocop#3095]
## Expected behavior

No warning.

## Actual behavior

    Warning: unrecognized parameter Style/AlignParameters:IndentationWidth found in /path/to/my/.rubocop.yml

**However, the cop works as expected.**

## Steps to reproduce the problem

Add config:

```yaml
Style/AlignParameters:
  EnforcedStyle: with_fixed_indentation
  IndentationWidth: 4
```

Run rubocop

## RuboCop version

```
$ rubocop -V
0.39.0 (using Parser 2.3.0.7, running on ruby 2.3.0 x86_64-darwin14)
```

**Comment by mikegee** [comment]
Yup, `IndentationWidth` isn't a parameter:

```
% bin/rubocop --show-cop Style/AlignParameters
# Supports --auto-correct
Style/AlignParameters:
  Description: Align the parameters of a method call if they span more than one line.
  StyleGuide: https://github.com/bbatsov/ruby-style-guide#no-double-indent
  Enabled: true
  EnforcedStyle: with_first_parameter
  SupportedStyles:
  - with_first_parameter
  - with_fixed_indentation
```

I'm not sure why you would expect no warning when providing an unrecognized parameter.

**Comment by donv** [comment]
@mikegee Thanks for responding.

The `IndentationWidth` parameter has effect through the `AutocorrectAlignment` mixin, so it is actually a parameter, although maybe not officially.

I would very much like it to become official.

BTW:  Where are the cop parameters documented?

**Comment by donv** [comment]
OK, found docs here:  https://github.com/bbatsov/rubocop/blob/master/config/default.yml

Would be nice to add `IndentationWidth` as a parameter to `Style/AlignParameters` cop.

**Comment by jfelchner** [comment]
@alexdowad I don't know if this is the same problem or not, but ever since `0.40.0` I'm now getting:

**Comment by jfelchner** [comment]
nm.  I see what the problem is.  I'll submit a PR for it.

**Comment by donv** [comment]
Thanks!

**Comment by faelin** [comment]
Same issue for `Layout/ExtraSpacing:AllowBeforeTrailingComments` — causes a warning even though it has a default value.

**Comment by dvandersluis** [comment]
@faelin I cannot reproduce, what warning are you seeing and what version are you using?

**Comment by faelin** [comment]
It was apparently a versioning problem.
